drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/data_warehouse_api.rb

```diff
@@ -0,0 +1,100 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisExternalApis::AcHmis
+  class DataWarehouseApi
+    SYSTEM_ID = 'ac_hmis_warehouse'.freeze
+
+    def self.enabled?
+      ::GrdaWarehouse::RemoteCredential.active.where(slug: SYSTEM_ID).exists?
+    end
+
+    # Returns the DW Client (Golden Record), for the specified mci_unique_id.
+    def golden_client_by_mci_unique_id(mci_unique_id)
+      conn.get("client/#{mci_unique_id}")
+        .then { |r| handle_error(r) }
+    end
+
+    # Returns the client records, for all DW sources, for the specified
+    # mci_unique_id.
+    def clients_by_mci_unique_id(mci_unique_id)
+      conn.get("clients/sources/#{mci_unique_id}")
+        .then { |r| handle_error(r) }
+    end
+
+    # Returns the DW Client (Golden Record), for the specified SrcSysKey/ClientId.
+    def client_by_client_id(client_id)
+      conn.get("client/#{src_sys_key}/#{client_id}")
+        .then { |r| handle_error(r) }
+    end
```

Comment:
We have no plans to use these endpoints, but I think it is fine to leave them.

Reply:
Sounds good. That's why I mostly didn't add tests. I tested one since it was an easier first step, being non-paginated.